### PR TITLE
Add extended view support to REST API

### PR DESCRIPTION
**Pull request by wmarshall484**

### CODE REVIEW

File: python/packages/streamsx/rest_primitives.py [hunk 1]
```diff
@@ -50,17 +63,76 @@ def get_instance(self):
     def get_job(self):
         return Job(self.rest_client.make_request(self.job), self.rest_client)
 
+    def stop_data_fetch(self):
+        self._stop.set()
+
+    def start_data_fetch(self):
+        self._stop.clear()
+        t = threading.Thread(target=self)
```
Comment (ddebrunner) [inline python/packages/streamsx/rest_primitives.py]: So `View` is a thread and we create a thread here? Just unclear what is going on.

**Comment by ddebrunner** [comment]
What's happening to the ` multi_graph_every` functions? Just being deleted or will they be recreated elsewhere?

**Comment by wmarshall484** [comment]
I'll recreate them elsewhere, but they definitely don't belong in rest.py. I'm thinking about where to put them. Since these were created with Jupyter visualization in mind, maybe something like `streamsx.jupyter` or `streamsx.jupyter_viz`

**Comment by ddebrunner** [comment]
I guess for backwards compat we just say they have moved, since they probably didn't have widespread usage yet.

**Comment by wmarshall484** [comment]
In commit [032694e](https://github.com/IBMStreams/streamsx.utility/pull/26/commits/032694e647fb1ebb898deadc396bb1af563c2b1d) I separated the threading logic from the view object. Hopefully it should be more clear.

**Comment by ddebrunner** [comment]
Will the topology api code call into the rest code to obtain the vcap services info, rather than having two copies of the code?

**Comment by wmarshall484** [comment]
@ddebrunner, that's a good question. Having the topology api code call the rest code to obtain the vcap services info reduces duplication at the cost of higher coupling.

Right now the topology api only depends on the rest api when views are present. If we make this change, the topology api will depend on the rest api whenever vcap services are supplied.

On the other hand, it is a lot of duplicated code.

**Pull request merged**